resume-buddy/src/main/java/com/google/sps/servlets/RevieweeDataServlet.java

```diff
@@ -31,11 +31,24 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String fname = getParameter(request, "fname", "");
     String lname = getParameter(request, "lname", "");
     String email = getParameter(request, "email", "");
-    reviewee = new Reviewee(fname, lname, email);
+    String school = getParameter(request, "school", "");
```

Comment:
Just FYI, you'll run into a bunch of merge conflicts because of your open PR that puts `getParameter` into a utility class.